### ISSUE TITLE
Need to control inputValue

### ISSUE THREAD

**Issue by Elfayer** [viclafouch/mui-chips-input#5]
**Is your feature request related to a problem? Please describe.**
I want to have a scan input, for that I use a debounce on the `onInputChange`, when 300ms passed, I set the chips. But after that, I need to reset the input value (but I do not have control over `inputValue`, see: https://github.com/viclafouch/mui-chips-input/blob/main/src/components/TextFieldChip/TextFieldChips.tsx#L66)

**Describe the solution you'd like**
Need to have `inputValue` as props of `MuiChipsInput`.

**Code usage sample**
```tsx
export default function ScanForm() {
  const [chips, setChips] = useState<string[]>([])
  const [scanValue, setScanValue] = useState('')
  const scanValueDebounced = useDebounce(scanValue, 300)

  useEffect(() => {
    setChips([...chips, scanValueDebounced])
    // reset scanValue, should reflect in MuiChipsInput
  }, [scanValueDebounced])

  function handleSubmit(e: React.FormEvent<HTMLFormElement>) {
    e.preventDefault()
    // ...
  }

  return (
    <form onSubmit={handleSubmit}>
      <MuiChipsInput value={chips} onInputChange={(v) => setScanValue(v)} />
      <Button>submit</Button>
    </form>
  )
}
```

**Comment by viclafouch** [comment]
Hey @Elfayer !

Fixed in 1.3.1 : https://github.com/viclafouch/mui-chips-input/releases/tag/v1.3.1

Ty 😘

**Comment by Elfayer** [comment]
Seems like I'm getting an error, and it is not working as expected. 🤔 
```
Warning: React does not recognize the `inputValue` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `inputvalue` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```
It's also not working as expected, see: https://codesandbox.io/s/mui-chips-input-forked-tysr0t?file=/src/App.tsx
When I `setScanValue("")` with prop `inputValue={scanValue}`, it doesn't update the input value.
